nx-X11/lib/X11/SetLocale.c

```diff
@@ -118,7 +118,7 @@ _Xsetlocale(
 
 #else /* X_LOCALE */
 
-#ifdef __DARWIN__
+#if defined(__APPLE__) || defined(__CYGWIN__)
```

Comment:
Shouldn't we stick to using __DARWIN__ here? Or do the Imakefiles somewhere also define __APPLE__?